lib/src/posthog_flutter_web_handler.dart

```diff
@@ -1,107 +1,180 @@
-// ignore: deprecated_member_use
-import 'dart:js';
+import 'dart:js_interop';
```

Comment:
do we need to bump the min. version of dart and flutter here?
https://dart.dev/interop/js-interop/package-web#package-web-vs-dart-html
https://github.com/PostHog/posthog-flutter/blob/e44de573ea0aa2068d7c33a839d2c12130b1887c/pubspec.yaml#L10-L11